lightning/src/ln/channel.rs

```diff
@@ -1583,6 +1543,17 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 		}, channel_monitor))
 	}
 
+	fn make_channel_static_info(&self) -> ChannelStaticInfo {
```

Comment:
Right now we create this on the fly.  In this PR or a followup we can make this a sub-structure of `Channel`.

Reply:
Good, encapsulation of Channel fields was discussed and favored before.

Reply:
This is now a sub-structure of `Channel` and this fn is eliminated.